Eevee.Sleep.Bot/Handlers/InteractionHandler.cs

```diff
@@ -116,4 +117,28 @@ await interaction
                 throw new ArgumentException($"Unhandled modal ID: {modalId}");
         }
     }
+
+    private static async Task OnButtonClicked(SocketMessageComponent component) {
+        var customId = component.Data.CustomId;
+        if (!customId.StartsWith("role")) {
```

Comment:
Refactor that `"role"` to a string info serializer class (I remember I commented this somewhere else, couldn't find it)

Reply:
I'm even less confident than usual here
Couldn't make it a generic seriarizer
I need advice 😔

[042bf5a](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/042bf5ae19def3a76b4fa219af2021bdffdd459e)

> (I remember I commented this somewhere else, couldn't find it)

this?
https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25#discussion_r1606038249

Reply:
Yeah that one

Basically this will be static class that has something like 
- `RoleButtonInteractionIdSerializer.serialize(id)` -> `role123456789 (string)`
- `RoleButtonInteractionIdSerializer.deserialize(text)` -> `123456789 (ulong)`

Returns can be changed to whatever you think it's good.

Reply:
If it doesn't have to be generic, I guess I could just rename the current implementation

Reply:
I thought I saw you have something serializer available already, isn't it usable here?

Reply:
This has already been resolved in the commit above, so I will leave this conversation as resolved!